3303Elevator/src/app/ElevatorSubsystem/Elevator/ElevatorButton.java

```diff
@@ -0,0 +1,18 @@
+package app.ElevatorSubsystem.Elevator;
+
+public class ElevatorButton {
+
+	private boolean status;
+
+	public ElevatorButton(){
+		this.status = false;
+	}
+
+	public boolean getStatus(){
+		return this.status;
+	}
+
+	public void press(){
+		this.status = true;
```

Comment:
Will be permanently pressed once this is called. Should have some way to turn it off